### PR TITLE
feat: added Texture Options to spritesheet loader

### PR DESCRIPTION
**Pull request by sam007mac**
##### Description of change

I've added the ability to set TextureOptions on sprites loaded via a spritesheet, as outlined in this Feature Request: #11148 

##### Pre-Merge Checklist

- [x] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)

I was having difficulty with the test suite but I tested my changes locally and it functioned as intended with normal and multipacked spritesheets.

### CODE REVIEW

File: src/spritesheet/spritesheetAsset.ts [hunk 1]
```diff
@@ -148,7 +158,16 @@ export const spritesheetAsset = {
             {
                 const imagePath = copySearchParams(basePath + (imageFilename ?? asset.meta.image), options.src);
 
-                const assets = await loader.load<Texture>([imagePath]);
+                let assets;
+
+                if (textureOptions)
```
Comment (GoodBoyDigital) [inline src/spritesheet/spritesheetAsset.ts:163]: I don't think we even need the `if (textureOptions)` check here as textureOptions can be null:

replacing the original line
`assets = await loader.load<Texture>([imagePath]);`

with
`assets = await loader.load<Texture>([{ src: imagePath, data: textureOptions}]);`

should work!

**Review by GoodBoyDigital — COMMENTED** [review]
thanks so much for this @sam007mac ! tiny tweak requested!

**Review by bigtimebuddy — COMMENTED** [review]
A test here would be useful to make sure the options are actually set.

File: src/spritesheet/spritesheetAsset.ts (hunk 1 above)
Comment (sam007mac) [reply src/spritesheet/spritesheetAsset.ts]: It looks like that check is still needed, removing it caused the SVG spritesheet test to fail. 
I could modify the SVG script to fix that issue but I don't know if that's in the scope of this PR, what are your thoughts?

**Comment by sam007mac** [comment]
I've added a test for checking the texture options.
I've also fixed the issue causing loadSVG to fail because of my change but I don't know if that should be under a separate PR or issue report, it's only partly related.

**Review by bigtimebuddy — APPROVED** [review]
Good fix! Thank you for the test.

**Review by GoodBoyDigital — APPROVED** [review]
thanks @sam007mac 👍

**Pull request merged**